tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/cluster/failover/LiveToLiveFailoverTest.java

```diff
@@ -207,6 +207,9 @@ public void scaleDownDelay() throws Exception {
 
       crash(session);
 
+      // Wait for failover to happen
+      Thread.sleep(100);
```

Comment:
it would be nice if we could instead use a class called Wait, that would keep trying until a certain condition was reached in the server.

This type of sleep will certainly eventually fail.

i'm looking for a case where the test always pass.. with this we will still have eventually a failure.

Reply:
Can you suggest a condition that can be used to check if ScaleDownHandler has kicked in?

Reply:
Fixed, thanks